R/adjective.R

```diff
@@ -88,6 +88,7 @@ adjective <- c(
   "primo",
   "proper",
   "rad",
+  "rockandroll",
```

Comment:
Sure, have hopefully fixed it - however, the CI system is complaining, maybe for reasons external to this change.